tests/tests.c

```diff
@@ -1678,12 +1678,15 @@ bool sirtest_filesystem(void) {
     }
 
     /* checking file descriptors. */
-    static const int bad_fds[] = {
+    static int bad_fds[] = {
         0,
         1,
         2,
         1234
     };
+    if (sirtest_get_wineversion()) { //-V547
```

Comment:
What's 547?

Reply:
Always false, since it will never be true (on Unix).